### DISCUSSION TITLE
permission denied for table http_request_queue

### DISCUSSION THREAD

**Discussion by begank** [supabase/supabase#29970] [Questions]
i got this error when i use a supabase client and try to select from net.http_request_queue(via api key) 
 and i have Exposed schemas NET 
{'code': '42501', 'details': None, 'hint': None, 'message': 'permission denied for table http_request_queue'}

**Comment by GaryAustin1** [comment]
That schema and its tables has not been granted permissions to the API users.  Normally though you should not modify schemas you don't control so maybe a better solution is to use a postgres security definer function to do what you want.
